src/coord.jl

```diff
@@ -1 +1,157 @@
 const Coord = SVector{4, Float64}
+const Coord234 = Union{SVector{2, Float64}, SVector{3, Float64}, SVector{4, Float64}}
+
+mutable struct Transformation <: CoordinateTransformations.Transformation
+    pj::Ptr{PJ}
+    function Transformation(pj::Ptr{PJ})
+        trans = new(pj)
+        finalizer(trans) do trans
+            trans.pj = proj_destroy(trans.pj)
+        end
+        return trans
+    end
+end
+
+function Transformation(
```

Comment:
it'll be nice if this function has a docstring, maybe describing (or giving examples of) the kinds of strings that are valid for `source_crs` and `target_crs`.

Reply:
Yeah good point, see 5212ae01c67bb38e757d69bbe196d4b398f79b5a